src/kaleidoscope_internal/LEDModeManager.h

```diff
@@ -406,9 +405,9 @@ class LEDModeManager {
    LEDModeManager::setupLEDMode(&::PLUGIN);
 
 #define _INIT_LED_MODE_MANAGER(...)                                            \
-   namespace kaleidoscope {                                             __NL__ \
-   namespace internal {                                                 __NL__ \
-   namespace led_mode_management {                                      __NL__ \
+   namespace kaleidoscope { /* NOLINT(build/namespaces) */              __NL__ \
+   namespace internal { /* NOLINT(build/namespaces) */                  __NL__ \
+   namespace led_mode_management { /* NOLINT(build/namespaces) */       __NL__ \
```

Comment:
All of these namespaces are inside a macro, we don't need to care whether the namespaces are fully declared prior to evaluating the macro. Tell cpplint to not warn about these.